internal/controller/driver_controller.go

```diff
@@ -1259,3 +1299,55 @@ func mergeDriverSpecs(dest, src *csiv1a1.DriverSpec) {
 		dest.CephFsClientType = src.CephFsClientType
 	}
 }
+
+func (r *driverReconcile) logrRotateContainer(imagePullPolicy corev1.PullPolicy, csiComponentName string, resources *corev1.ResourceRequirements) corev1.Container {
```

Comment:
Please have the container definition inline like any other container already defined.

Reply:
I dont see any other common container function